[Transform][DOCS]rewrite client ip example to use continuous transform (#49822)
elastic/elasticsearch: docs/reference/transform/examples.asciidoc

----------------------------------
// TEST[skip:setup kibana sample data]

<1> This is the destination index for the {transform}.

> <2> This configures the {transform} in continuous mode using the field `timestamp`

Reviewer: Not mandatory, but I don't see that we talk about "continous mode" in the docs so this might be clearer:
```suggestion
<2> Configures the {transform} to run continuously. It uses the `timestamp` field
```

> as syncronization field. The `delay` is set to 60s and represents the worst case

Reviewer: ```suggestion
to synchronize the source and destination indices. We expect the worst case
```
The time delay between the current time and the latest input data time.

> ingest delay we expect.

Reviewer: ```suggestion
ingestion delay is defined as 60 seconds.
```

<3> The data is grouped by the `clientip` field.
<4> This `scripted_metric` performs a distributed operation on the web log data
to count specific types of HTTP responses (error, success, and other).
<5> This `bucket_script` calculates the duration of the `clientip` access based
on the results of the aggregation.

> After creation the transform can be started with:

Reviewer: ```suggestion
After you create the  {transform}, you must start it:
```

[source,console]
----------------------------------
POST _transform/suspicious_client_ips/_start
----------------------------------
// TEST[skip:setup kibana sample data]

> Shortly after the first results should be available using search on the destination

Reviewer: ```suggestion
Shortly thereafter, the first results should be available in the destination
```

index:

[source,console]
----------------------------------
GET sample_weblogs_by_clientip/_search
----------------------------------
// TEST[skip:setup kibana sample data]

The search result shows you data like this for each client IP:

[source,js]
----------------------------------
    "hits" : [
      {
        "_index" : "sample_weblogs_by_clientip",
        "_id" : "MOeHH_cUL5urmartKj-b5UQAAAAAAAAA",
        "_score" : 1.0,
        "_source" : {
          "geo" : {
            "src_dc" : 2.0,
            "dest_dc" : 2.0
          },
          "clientip" : "0.72.176.46",
          "agent_dc" : 2.0,
          "bytes_sum" : 4422.0,
          "responses" : {
            "total" : 2.0,
            "counts" : {
              "other" : 0,
              "success" : 2,
              "error" : 0
            }
          },
          "url_dc" : 2.0,
          "timestamp" : {
            "duration_ms" : 5.2191698E8,
            "min" : "2019-11-25T07:51:57.333Z",
            "max" : "2019-12-01T08:50:34.313Z"
          }
        }
      }
    ]
----------------------------------
// NOTCONSOLE

> NOTE: Like other kibana sample data sets, the web log sample dataset contains

Reviewer: ```suggestion
NOTE: Like other Kibana sample data sets, the web log sample dataset contains
```

> timestamps relative to when they are installed, including timestamps in the future.

Reviewer: ```suggestion
timestamps relative to when you installed it, including timestamps in the future.
```

> The continuous {transform} will pickup the data points once they are in the past

Reviewer: ```suggestion
The {ctransform} will pick up the data points once they are in the past
```

> (see delay). This enables you the see continuous mode in action, although we are

Reviewer: > (see delay)

I don't feel like referring back to the 60 second delay is enough of an explanation.  I suggest omitting this bit.  Otherwise, it needs more explanation.

Author: I removed the remarks about delay. Let me know if I got this wrong.
